### DISCUSSION TITLE
Stamp is not visible in signed files with pyhanko CLI

### DISCUSSION THREAD

**Discussion by regonzalom** [MatthiasValvekens/pyHanko#272] [Q&A] [answered]
Hi people,

I'm signing PDF documents successfully but signed pdf don't have a stamp in it's body.

![image](https://github.com/MatthiasValvekens/pyHanko/assets/1321510/dd6f225c-13e0-443f-ac52-f9d46176954f)

I'm signing the document with this call:

`'pyhanko --config /home/bitnami/pyhanko/pyhanko.yml sign addsig --field firmaColaborador --name "Noelia Delgado" --reason "All OK" --timestamp-url "https://freetsa.org/tsr" --style-name default pemder --key /home/bitnami/certificates/noelia_key.pem --cert /home/bitnami/certificates/noelia_cert.pem /home/bitnami/resi_documents/documento.pdf /home/bitnami/resi_documents/documento_signed.pdf --no-pass'`

And this is my yml config document:

`stamp-styles:
  default:
    type: text
    stamp-text: "firma"
    border-width: 0`

Am I doing something wrong? Thanks,
Gonzalo

**Comment by MatthiasValvekens** [comment]
Hi @regonzalom 

In order to create a visible signature in a new signature field, you have to tell pyHanko where to put it on the page. That's done with the `--field PAGE/X1,Y1,X2,Y2/NAME` syntax instead of `--field NAME`. But I agree that it's not fully clear in the documentation (this bit is only explained in the section about `addfields`), and the CLI should probably throw up a useful error message if you use `--style-name` with an invisible signature field spec. I'll see what I can do about that.

**Reply by MatthiasValvekens** [reply, marked as answer]
See #273. This invocation will now fail with an error that tells you what to do :).

**Reply by regonzalom** [reply]
Thanks @MatthiasValvekens your tips did the tric. 

But I still have something wrong in my implementation. I getting a standard signature appearance despite I'm using a custom signature style.

This is my CLI invocation:

`pyhanko --config /home/bitnami/pyhanko/pyhanko.yml sign addsig --field 1/10,10,100,100/ok --name "DELGADO NOELIA" --reason "Firma OK" --timestamp-url "https://freetsa.org/tsr" pemder --key /home/bitnami/certificates/5bf50252-091e-11ee-acea-02eafd9a22b7_key.pem --cert /home/bitnami/certificates/5bf50252-091e-11ee-acea-02eafd9a22b7_cert.pem /home/bitnami/resi_documents/e1b3add3-5b8e-403d-97e7-f36718ae8d83.pdf /home/bitnami/resi_documents/e1b3add3-5b8e-403d-97e7-f36718ae8d83_signed.pdf --no-pass
` 
And this is my pyhanko config file:

![image](https://github.com/MatthiasValvekens/pyHanko/assets/1321510/7a329b34-a715-486d-960d-77bbd2fe812f)

Am I making a mistake? Thank you.
Gon

**Reply by MatthiasValvekens** [reply]
I don't see the `--style-name` argument in your new CLI invocation. The `NAME` in `--field PAGE/X1,Y1,X2,Y2/NAME` is the name of the signature field, not that of the style. :)

**Reply by regonzalom** [reply]
@MatthiasValvekens you did it again :-D... and you answered me in hours... It's awesome!

Thank you for your answers and for this amazing tool.
Regards
